apps/src/code-studio/pd/workshop_dashboard/components/survey_results/single_choice_responses.jsx

```diff
@@ -14,7 +14,11 @@ export default class SingleChoiceResponses extends React.Component {
   render() {
     let totalAnswers = Object.values(this.props.answers).reduce((sum, x) => sum + x, 0);
 
-    let otherAnswers = _.difference(Object.keys(this.props.answers), this.props.possibleAnswers);
+    // The split is needed for scale questions. The top and bottom responses have text
+    // like "1 - not ready / 5 - very ready" and we need to extract the number
+    let possibleAnswers = this.props.answerType === 'scale' ? this.props.possibleAnswers.map((x) => x.split(' ')[0]) : this.props.possibleAnswers;
```

Comment:
Does possibleAnswers contain the numbers within the scale range (2-4 in this example)?

Reply:
Yes